### PR TITLE
Add band arithmetic and raster calculator tools!

### PR DESCRIPTION
**Pull request by DanielJDufour**

### CODE REVIEW

File: src/components/tools/raster-calculator/raster-calculator.container.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  raster: state.raster
+});
+
+const mapDispatchToProps = dispatch => ({
+  run_raster_calculator: (raster, raster_calculator) => {
+    if (!raster) return dispatch(show_alert('Please add a raster before running this tool.'));
+    if (!raster_calculator) return dispatch(show_alert('Please add a raster calculator operation before running this tool.'));
+    try {
+      dispatch(start_loading('Running Raster Calculator'));
+      console.log("raster_calculator:", raster_calculator);
```
Comment (sdpeyton) [inline src/components/tools/raster-calculator/raster-calculator.container.js]: stray console log

File: src/services/ToolListService.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ let ToolListService = {
   search_tool_list(value) {
     return new Promise((resolve, reject) => {
       this.get_tool_list().then(tool_list => {
+        console.log("tool_list:", tool_list);
```
Comment (sdpeyton) [inline src/services/ToolListService.js]: stray console log

**Comment by sdpeyton** [comment]
I think we need some new icons here: 
screen shot 2018-05-13 at 8 44 23 am

File: src/components/tools/raster-calculator/raster-calculator.container.js (hunk 1 above)
Comment (DanielJDufour) [reply src/components/tools/raster-calculator/raster-calculator.container.js]: shucks. I'll fix that

**Pull request merged**